server/src/services/DiscordClient.js

```diff
@@ -129,17 +129,23 @@ class DiscordClient {
       webhooks: new Set(),
       scanner: new Set(),
       blockedGuildNames: new Set(),
+      rules: new Map(),
     }
     const scanner = config.getSafe('scanner')
+    perms.scannerCooldowns = {}
+
     try {
       const guilds = user.guilds.map((guild) => guild.id)
       if (this.strategy.allowedUsers.includes(user.id)) {
         Object.keys(this.perms).forEach((key) => (perms[key] = true))
         perms.admin = true
         config.getSafe('webhooks').forEach((x) => permSets.webhooks.add(x.name))
-        Object.keys(scanner).forEach(
-          (x) => scanner[x]?.enabled && permSets.scanner.add(x),
-        )
+        Object.keys(scanner).forEach((x) => {
+          if (scanner[x]?.enabled) {
+            permSets.scanner.add(x)
+            scanner[x].userCooldownSeconds = 0
```

Comment:
```suggestion
            perms.scannerCooldowns[x] = 0
```
We want to assign this value to the local perms obj, not the config ref.